application/src/main/java/org/thingsboard/server/service/mail/TbMailSender.java

```diff
@@ -151,8 +166,8 @@ public void refreshAccessToken() throws ThingsboardException {
                 tokenExpires = System.currentTimeMillis() + (tokenResponse.getExpiresInSeconds().intValue() * 1000);
             }
         } catch (Exception e) {
-            log.warn("Unable to retrieve access token: {}", e.getMessage());
-            throw new ThingsboardException("Error while retrieving access token: " + e.getMessage(), ThingsboardErrorCode.GENERAL);
+            log.error("Unable to retrieve access token: {}", e.getMessage());
+            throw new RuntimeException("Error while retrieving access token: " + e.getMessage());
```

Comment:
Why is not MessagingException expected from the super method signature? 
Did you notice the difference in behavior on UI?